layers/core_checks/cc_image_layout.cpp

```diff
@@ -401,19 +401,19 @@ bool CoreChecks::ValidateLayoutVsAttachmentDescription(RenderPassCreateVersion r
         if (use_rp2 && ((first_layout == VK_IMAGE_LAYOUT_DEPTH_STENCIL_READ_ONLY_OPTIMAL) ||
                         (first_layout == VK_IMAGE_LAYOUT_SHADER_READ_ONLY_OPTIMAL) ||
                         (first_layout == VK_IMAGE_LAYOUT_DEPTH_READ_ONLY_STENCIL_ATTACHMENT_OPTIMAL))) {
-            skip |= LogError(device, "VUID-VkRenderPassCreateInfo2-pAttachments-02522",
-                             "vkCreateRenderPass2(): Cannot clear attachment %d with invalid first layout %s.", attachment,
-                             string_VkImageLayout(first_layout));
+            skip |= LogError("VUID-VkRenderPassCreateInfo2-pAttachments-02522", device, layout_loc,
+                             "(%s) is an invalid for pAttachments[%d] (first attachment to have LOAD_OP_CLEAR).",
```

Comment:
```suggestion
                             "(%s) is an invalid for pAttachments[%" PRIu32 "] (first attachment to have LOAD_OP_CLEAR).",
```

and I saw other %d, I guess they could all be turned into %" PRIu32 "

Reply:
this is honestly just worth making a "destroy all `%d` and `%u` left" PR... I really don't know why we have't done it sooner